kahuna/public/js/components/gr-collections-panel/gr-collections-panel-node.html

```diff
@@ -19,31 +19,48 @@
         </button>
 
         <a class="node__name flex-spacer"
-           ui:sref="search.results({query: (node.data.name)})">
+           ui:sref="search.results({query: ctrl.getCollectionQuery(node.data.content.pathId)})">
             {{node.data.name}}
         </a>
 
+
         <button class="node__action clickable" type="button"
                 ng:if="ctrl.deletable"
-                ng:click="ctrl.remove()">
+                ng:show="ctrl.editing"
```

Comment:
@akash1810 that's to disable inputs? I think we're talking about showing / hiding content.